### PR TITLE
release 2.0.0

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -148,5 +175,7 @@
 - Confine rspec pinning to ruby 1.8
 
 
-
 \* *This Change Log was automatically generated by [github_changelog_generator](https://github.com/skywinder/Github-Changelog-Generator)*
+
+
+\* *This Changelog was automatically generated by [github_changelog_generator](https://github.com/skywinder/Github-Changelog-Generator)*
```
Comment (raphink) [inline CHANGELOG.md]: That's two footers :wink:

File: CHANGELOG.md [hunk 2]
```diff
@@ -17,6 +43,7 @@
 
 **Merged pull requests:**
 
+- release 1.1.1 [\#56](https://github.com/camptocamp/puppet-systemd/pull/56) ([bastelfreak](https://github.com/bastelfreak))
```
Comment (raphink) [inline CHANGELOG.md]: I don't know if gcg could ignore this one

File: CHANGELOG.md (hunk 1 above)
Comment (bastelfreak) [reply CHANGELOG.md]: fixed

File: CHANGELOG.md (hunk 2 above)
Comment (bastelfreak) [reply CHANGELOG.md]: fixed

**Comment by bastelfreak** [comment]
I would like to include https://github.com/camptocamp/puppet-systemd/pull/71 before we release

**Comment by fatmcgav** [comment]
@bastelfreak / @raphink What are the chances of getting a 2.0.0 release?

**Comment by bastelfreak** [comment]
@fatmcgav I would love to make a new release, but I think we need to solve https://github.com/camptocamp/puppet-systemd/pull/73 before.

**Comment by fatmcgav** [comment]
@bastelfreak OK, sounds like a plan...

**Comment by bastelfreak** [comment]
I rebased the branch and updated the CHANGELOG.md. @raphink @mcanevet can we please get a new release?

File: CHANGELOG.md (hunk 1 above)
Comment (alexjfisher) [reply CHANGELOG.md]: @bastelfreak  https://github.com/camptocamp/puppet-systemd/pull/78

**Pull request merged**